pom.xml

```diff
@@ -131,6 +131,7 @@
       <groupId>com.infradna.tool</groupId>
       <artifactId>bridge-method-annotation</artifactId>
       <version>1.14</version>
+      <optional>true</optional>
```

Comment:
or scope provided?

Reply:
Either seem to work pretty much the same.